### PR TITLE
Add text labels to video bookmarks ("Rename Bookmark")

### PR DESCRIPTION
**Pull request by Korlawulki**
Adding names as labels to bookmarks (new "Rename Bookmark" menu item), making bookmark handling easier. Also, confirmation dialogs are shown when removing bookmarks (Remove? -> Yes/No).

Detailed description:
This  patch adds a menu item "Rename Bookmark" to the context menu of bookmarks. Everything entered here is stored in the bookmark database table (schema changed!). Instead of the standard labels (e.g. "Bookmark 1 (01:23:45)"), the stored text label for each bookmark is show as e.g. "Action Scene (00:56:01)". If no text label has been entered, the old default label is shown.

Very useful if someone has videos of rock concerts. Now, it's possible to give each bookmark the name of the title of the song.

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
please do not use any string ID below 10000.

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -18319,3 +18319,18 @@ msgstr ""
...
+msgstr ""
+
+#: xbmc/dialogs/GUIDialogVideoBookmarks.cpp
+msgctxt "#38043"
+msgid "Removing all bookmarks"
+msgstr ""
+
+#: xbmc/dialogs/GUIDialogVideoBookmarks.cpp
+msgctxt "#38044"
+msgid "Removing bookmark"
```
Comment (un1versal) [inline addons/resource.language.en_gb/resources/strings.po]: Removing or Remove?

Comment (HitcherUK) [inline addons/resource.language.en_gb/resources/strings.po]: Or 'Removed...' if the action has already been preformed and this is simply telling you so with the OK dialog.

**Comment by Korlawulki** [comment]
It's used for the confirmation request dialog. So, the process of removal is formally started and in progress.
Another proposal: "About to remove ..."

File: addons/resource.language.en_gb/resources/strings.po (hunk 1 above)
Comment (Korlawulki) [inline addons/resource.language.en_gb/resources/strings.po]: It's used for the confirmation request dialog. So, the process of removal is formally started and in progress.
Another proposal: "About to remove ..."

Comment (un1versal) [inline addons/resource.language.en_gb/resources/strings.po]: In that case Removing is better also less to translate and shorter translated string

Comment (HitcherUK) [inline addons/resource.language.en_gb/resources/strings.po]: So this is in the OK dialog? If so, and due to the fact there's no way to cancel, I think 'Removed...' is better.

Comment (Korlawulki) [inline addons/resource.language.en_gb/resources/strings.po]: The action can be cancelled by the user.

Comment (HitcherUK) [inline addons/resource.language.en_gb/resources/strings.po]: Thanks for the confirmation - 'Removing...' it is then.

**Comment by Rechi** [comment]
@Korlawulki this needs a rebase

**Comment by MartijnKaijser** [comment]
I will close this PR. If there's interest please open a new one (with required changes if needed).

**Pull request closed without merging**